source/ranges.tex

```diff
@@ -3365,21 +3365,22 @@
 
 \begin{codeblock}
 namespace std::ranges {
-  template<@\libconcept{copy_constructible}@ W, @\libconcept{semiregular}@ Bound = unreachable_sentinel_t>
+  template<@\libconcept{move_constructible}@ W, @\libconcept{semiregular}@ Bound = unreachable_sentinel_t>
     requires (is_object_v<W> && @\libconcept{same_as}@<W, remove_cv_t<W>> &&
               (@\exposid{is-integer-like}@<Bound> || @\libconcept{same_as}@<Bound, unreachable_sentinel_t>))
   class @\libglobal{repeat_view}@ : public view_interface<repeat_view<W, Bound>> {
   private:
     // \ref{range.repeat.iterator}, class \tcode{range_view::\exposid{iterator}}
     struct @\exposid{iterator}@;    // \expos
 
-    @\exposid{copyable-box}@<W> @\exposid{value_}@ = W();   // \expos, see \ref{range.copy.wrap}
+    @\exposid{movable-box}@<W> @\exposid{value_}@ = W();    // \expos, see \ref{range.move.wrap}
     Bound @\exposid{bound_}@ = Bound();    // \expos
 
   public:
     repeat_view() requires @\libconcept{default_initializable}@<W> = default;
 
-    constexpr explicit repeat_view(const W& value, Bound bound = Bound());
+    constexpr explicit repeat_view(const W& value, Bound bound = Bound())
+      requires @\libconcept{copy_constructible}@<W>;
```

Comment:
The corresponding change is missing in its detailed specifications.

Reply:
Thanks, I'll fix it later